businesscentral-indexing/image.yaml

```diff
@@ -59,10 +59,9 @@ modules:
                   ref: master
       install:
           - name: jboss.container.openjdk.jdk
-            version: "8"
+            version: "11"
```

Comment:
at least for indexing I think that would be better stay on java 8.
elasticsearch 5.6.5 does not support java 11: https://www.elastic.co/support/matrix#matrix_jvm